xc-admin/packages/xc-admin-common/src/__tests__/GovernancePayload.test.ts

```diff
@@ -1,7 +1,8 @@
 import { PublicKey, SystemProgram } from "@solana/web3.js";
-import { decodeExecutePostedVaa, decodeHeader } from "..";
+import { decodeExecutePostedVaa, decodeHeader, encodeHeader } from "..";
+import { encodeExecutePostedVaa } from "../governance_payload/ExecutePostedVaa";
 
-test("GovernancePayload", (done) => {
+test("GovernancePayload ser/de", (done) => {
```

Comment:
I don't quite like this test as changing a payload/structure requires spending time on fixing it here.

specially that you go from "decode" to "encode" again. it means on change either 
(1) someone should hand roll the struct manually (2) that someone should write down another structure and calls encode to get the new value.

I think it's better to encode and decode a struct. That's easier to modify/update.

Reply:
I refactored to to `struct -> encode -> bytes -> decode -> struct`. Should I delete the hardcoded bytes arrays and make it `struct -> encode -> decode -> struct`